### PR TITLE
Add handy helper for pages specific JavaScript

### PR DESCRIPTION
**Pull request by jonatanklosko**
Now adding JavaScript to a subset of pages is cleaner and more concise.
Examples:
- Actions
```javascript
onPage('users#index', function() {
  // Code for the index view action in users controller.
});
````
- Controllers
```javascript
onPage('competitions', function() {
  // Code for all views in competitions controller.
});
````
- Chaining
```javascript
onPage('competitions#index, users', function() {
  // Code for the index action view in competitions controller and all views in users controller.
});
````

### CODE REVIEW

File: WcaOnRails/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -284,3 +284,24 @@ $(function() {
...
+// Helpers
+
+// Executes the given function on the specified page/pages.
+// The given string should have a format: '<controller>#<action>'.
+// The action name preceded by '#' is optional.
+// Could be followed by comma and another pair.
+// Example: 'users#show, users#edit, users#update, competitions'.
+function onPage(controllersWithActions, fun) {
+  controllersWithActions = controllersWithActions.replace(/\s/g, '');
+  controllersWithActions = controllersWithActions.split(',');
```
Comment (timhabermaas) [inline WcaOnRails/app/assets/javascripts/application.js]: You might need to strip the whitespace after splitting at `,` for your example to work.

Comment (jonatanklosko) [inline WcaOnRails/app/assets/javascripts/application.js]: Why? I thought the previous line removes all white spaces.

Comment (timhabermaas) [inline WcaOnRails/app/assets/javascripts/application.js]: Oh, you're right, missed that.

**Comment by jfly** [comment]
I like this a lot, very simple and clean. :shipit: :rocket:

**Pull request merged**